### PR TITLE
[Test/NNFW] fix invalid tc

### PR DESCRIPTION
**Pull request by jaeyun-jung**
nnfw-1.10 was released and now supports dynamic dimension.
Enable testcases and fix invalid test conditions.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc [hunk 1]
```diff
@@ -205,13 +205,13 @@ TEST (nnstreamer_nnfw_runtime_raw_functions, DISABLED_set_dimension)
...
   for (int idx = 0; idx < tensor_size; idx++)
-    ((gint32 *) input.data)[idx] = idx;
+    ((gfloat *) input.data)[idx] = (gfloat) idx;
 
   ret = sp->invoke_NN (&prop, &data, &input, &output);
   EXPECT_EQ (ret, 0);
 
   for (int idx = 0; idx < tensor_size; idx++)
-    EXPECT_EQ (((gint32 *) output.data)[idx], idx + 2);
+    EXPECT_EQ (((gfloat *) output.data)[idx], (gfloat) (idx + 2));
```
Comment (dongju-chae) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc:214]: How about using EXPECT_FLOAT_EQ or EXPECT_DOUBLE_EQ

Comment (jaeyun-jung) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc]: updated :)

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc [hunk 2]
```diff
@@ -122,7 +122,7 @@ TEST (nnstreamer_nnfw_runtime_raw_functions, get_dimension)
  * @brief Set input dimensions with nnfw subplugin
  * @todo enable when nnfw_set_dimension API is interally implemented
```
Comment (gichan-jang) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc:123]: Now it is unnecessary?

**Review by gichan-jang — APPROVED** [review]
LGTM

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc (hunk 2 above)
Comment (jaeyun-jung) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc]: right!

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**